x-pack/plugins/observability/public/pages/slo_edit/components/slo_edit_form_objectives.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiFieldNumber,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiFormLabel,
+  EuiSelect,
+  EuiSpacer,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Control, Controller, UseFormWatch } from 'react-hook-form';
+import type { BudgetingMethod, CreateSLOParams } from '@kbn/slo-schema';
```

Comment:
Related to the previous comment about the BudgetingMethod type declaration. This should be exported from the rest_specs/
That does not change anything here though.